SeleniumTestXUnit/Tests/Features/Project/DragAndDrop.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Project Interacting
+    As a logged in user, the user should be able to interact with the project links to reorganize them.
+    
+    Background:
+        Given the user is logged in
+
+    @Regression @go.to.login.page @create.project @delete.project
```

Comment:
No need to have the @go.to.login.page hook unless you are going to reuse browser instances for the whole test execution. I don't recommend doing it that way yet.

Reply:
Done